### ISSUE TITLE
What is the purpose of these lines?

### ISSUE THREAD

**Issue by hiyali** [tidwall/evio#55]
https://github.com/tidwall/evio/blob/b353be3a765785dafabbb63a7b25aadec3533946/internal/socktoaddr.go#L28-L29

Thank you for the great framework! and I want to ask you about this question.

**Comment by thumbsized** [comment]
good catch, seems like tidwall wanted to put some logic in there but forgot :)
